src/components/DashboardAppBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const DashboardAppBar = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const logout = () => {
+    firebase.logout().then(() => {
+      navigate('/')
+    })
+  }
```

Comment:
It would be great to have tests for this. You can test two things:
1. It should call `firebase.logout()` function on `Logout` button click.
2. It should navigate to `/` on click.

Reply:
I think it will be good to make another task for configuration and first tests